sycl/include/CL/sycl/pipes.hpp

```diff
@@ -8,80 +8,11 @@
 
 #pragma once
```

Comment:
Why not remove this file completely? AFAIK users are not supposed to include this file directly anyway.